server/src/controllers/courseInstance.ts

```diff
@@ -71,7 +79,7 @@ export async function getCourseInstance(req: Request, res: Response): Promise<vo
     endDate: instance.endDate,
     teachingMethod: instance.teachingMethod as TeachingMethod,
     gradingType: instance.gradingType as GradingType,
-    responsibleTeacher: responsibleTeacher?.name ?? '-',
+    teachersInCharge: [],
```

Comment:
Can we fill in the teacher names here same way as in the `getAllCourseInstances` function?

Reply:
Good point, forgot about that.